serverless/serverless-architecture.adoc

```diff
@@ -6,3 +6,6 @@ include::modules/serverless-document-attributes.adoc[]
 
 include::modules/knative-serving.adoc[leveloffset=+1]
 include::modules/knative-cli.adoc[leveloffset=+1]
+
+== Knative Eventing
+A developer preview version of Knative Eventing is available for use with {ServerlessProductName}, however this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information, see the https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
```

Comment:
what about, "For more information, including install/uninstall instructions and samples, see the `https...`" just to spell out the specifics and avoid the earlier situation.  Or is that too concrete for a component that's not in TP?

Reply:
Makes good sense, thanks for the suggestion!

Reply:
For me it was good enough. A little more verbose will not hurt.